packages/bpk-component-scrollable-calendar/src/BpkScrollableCalendarGridList.tsx

```diff
@@ -123,7 +123,7 @@ const BpkScrollableCalendarGridList = (props: Props) => {
     monthItemHeights[index] || estimatedMonthItemHeight;
 
   const rowRenderer = ({ index, style }: { index: number; style: {} }) => (
-    <div style={style}>
+    <div className={getClassName('bpk-scrollable-calendar-grid-list__item')} style={style}>
```

Comment:
testing the class's only styling `display: inline-table;` I don't think it has the effect we expect anymore? Perhaps it's safe to remove?

Reply:
Thanks, removed